server/apparmor/apparmor.go

```diff
@@ -0,0 +1,146 @@
+package apparmor
```

Comment:
Since have already defined a `apparmor` build tag, maybe it's better to use here and add a `apparmor_unsupported.go` for those won't support this feature.